### ISSUE TITLE
Flashing not working

### ISSUE THREAD

**Issue by luukvankooten** [esp-rs/espflash#126]
```zsh
$ espflash /dev/cu.usbserial-0001 target/xtensa-esp32-espidf/release/led
Serial port: /dev/cu.usbserial-0001
Connecting...

Unable to connect, retrying with extra delay...
Unable to connect, retrying with default delay...
Unable to connect, retrying with extra delay...
Unable to connect, retrying with default delay...
Unable to connect, retrying with extra delay...
Unable to connect, retrying with default delay...

Error: espflash::timeout

  × Error while connecting to device
  ╰─▶ Timeout while running ReadReg command
```

Every single time I run espflash this output is sent to me. It looks like it's doing something because my previous program which is flashed stops. Looked into the issues, it is like #85 but not sure. I'm also on Mac M1 don't know if it has todo something with that. And other programs are successfully uploaded with the Arduino IDE.

**Comment by luukvankooten** [comment]
Got a workaround: install espflash with this specific comment: `cargo install espflash --git https://github.com/esp-rs/espflash.git --rev fd205c7d21cf7eb339ccb76d270d1821577c0254 --force`

**Comment by pyaillet** [comment]
I faced the same problem. Reverting to v1.1.0 solved it for me.

I can provide

**Comment by jessebraham** [comment]
Would either or both of you be able to check whether this issue persists using the newest version (`1.4.0`) please?

**Comment by pyaillet** [comment]
I switched to 1.4.0-dev last week to test panic backtrace of `espmonitor` and did not face any problems since then.
I will try with the `1.4.0` release though.

**Comment by pyaillet** [comment]
So I just installed the released `1.4.0` version and it works perfectly on my side.

Thank you for this update :)
